app/views/student_flashcards/show.html.erb

```diff
@@ -0,0 +1,16 @@
+show page for student flashcard
+
+<%= link_to 'Back', student_student_flashcard_sets_student_flashcards_path, class: "btn btn-outline-dark m-2" %>
+<div class="container mt-3">
+  <div class="quiz-question-card">
+   <div class="container">
+     <h1>Edit the student flashcard</h1>
+
+     <%= simple_form_for @student_flashcard do |f| %>
+       <%= f.input :question %>
```

Comment:
I'd add the answer. Are the original questions appearing? On edit they often do, but I'm not sure if it will here. If not, I'd set the initial value to be the ones of the template! But don't need to do this now for the merge, I guess it will be easier later